config.yaml

```diff
@@ -11,6 +11,18 @@ logging:
   file: ./data/ambianic-log.txt
   level: INFO
 
+# Store notifications provider configuration
+# see https://github.com/caronc/apprise#popular-notification-services for syntax examples
+# notifications:
+#   email:
```

Comment:
I do agree in part because we would loose flexibility. Currently we focus on a use case but users could appreciate this on different pipelines. Automation task could benefit of this approach for example. Can we try this approach and validate with end users first?

Reply:
Sounds good.

Reply:
Great thanks!